frame/proxy/src/lib.rs

```diff
@@ -305,17 +311,23 @@ pub mod pallet {
 		/// Weight is a function of the number of proxies the user has (P).
 		/// # </weight>
 		/// TODO: Might be over counting 1 read
-		#[pallet::weight(T::WeightInfo::anonymous(T::MaxProxies::get().into()))]
-		pub fn anonymous(
+		#[pallet::weight(T::WeightInfo::proxied_keyless(T::MaxProxies::get().into()))]
+		pub fn proxied_keyless(
```

Comment:
please avoid breaking changes

Reply:
Sure, this was more of a quick search and replace, I'll keep the previous "deprecated" method around. Any other things to consider when introducing this kind of API changes?

Reply:
I think we should rename the functions though... Are you suggesting we should add a new function with a new name?

AFAIK this "breaking change" would only be breaking for JS integrations, but all txs and signers will be okay

Reply:
rename it is breaking for JS and ledger. let's avoid that unless absolutely necessary.

and new parameter is a transaction_version breaking change.